recipes/node2vec/LICENSE

```diff
@@ -0,0 +1,9 @@
+MIT License
```

Comment:
Thanks for packaging the license. But please open an issue [upstream](https://github.com/eliorc/node2vec) asking them to include the license in the source distro in pypi.